iXBRLViewerPlugin/viewer/src/js/search.js

```diff
@@ -134,7 +134,12 @@ export class ReportSearch {
         );
     }
 
-
+    namespacesFilter(s, item) {
+        return (
+            s.namespacesFilter.length === 0 ||
+            s.namespacesFilter.some(p => item.getConceptPrefix() === p)
```

Comment:
It doesn't matter much, but it seems more natural for `namespacesFilter` to be a `Set` and to use `.has`?